### DISCUSSION TITLE
how can save both token twitter and discord in session with nextAuth?

### DISCUSSION THREAD

**Discussion by zenn37** [community/community#50365] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everybody, I'm working with nextAuth to authenticate with twitter and discord. But I have a suffering, I want to save both Twitter and discord tokens in the session. However, I only get token Twitter when login Twitter and discord is the same. 
Please let me know how to solve this problem. Thank you so much!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
